File: src/test/java/com/dotnomi/fabricdependencyinjection/testclasses/beanlist/AnalyticsPlugin.java
package com.dotnomi.fabricdependencyinjection.testclasses.beanlist;

import com.dotnomi.fabricdependencyinjection.annotation.ModScoped;

@ModScoped
public class AnalyticsPlugin implements Plugin {
  @Override
  public String getName() {
    return "Analytics";
  }
}

File: src/test/java/com/dotnomi/fabricdependencyinjection/kotlin/testclasses/beanlist/AnalyticsPlugin.kt
package com.dotnomi.fabricdependencyinjection.kotlin.testclasses.beanlist

import com.dotnomi.fabricdependencyinjection.annotation.ModScoped

@ModScoped
class AnalyticsPlugin() : Plugin {
    override fun getName(): String {
        return "Analytics"
    }
}
